src/pages/settings/[id].tsx

```diff
@@ -118,6 +118,13 @@ export default function SettingsPage() {
                 }
                 email={data.email ? data.email : "Missing Email"}
                 bio={data.profile?.bio ? data.profile.bio : "Missing Bio"}
+                dob={data.profile?.dateOfBirth ? data.profile.dateOfBirth : ""}
```

Comment:
No need to abbreviate, dateOfBirth is okay

Reply:
I have it as dob because we were using dateOfBirth for the state in EditProfile.tsx, so I needed a different identifier